packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -84,6 +75,30 @@ export const BooleanScalarInputType = new GraphQLInputObjectType({
   }
 })
 
+/**
+ * Custom GraphbackJSON scalar
+ */
+export const GraphbackJSONScalarInputType = new GraphQLInputObjectType({
+  name: getInputName(GraphbackJSON),
+  fields: {
+    ne: { type: GraphQLString },
+    eq: { type: GraphQLString },
+    contains: { type: GraphQLString }
```

Comment:
Curious to see how this filtering works for PG.

Reply:
It doesn't :( `contains` will need to go.

Reply:
It is more a problem with knex.js rather than pg right? I mean, we'll have to send the right json operator in the built query since knex cannot do that for us automatically as for a string?

Reply:
Knex does not map operators for querying JSON.

It's possible, but not a quick feature to add. [knexQueryMapper]() does not know the column type when mapping operators, and the syntax for querying JSON columns is completely different:

```
select comments->>'name' from note where (comments->'complete')::jsonb ? TRUE;
```

Reply:
Yes, this kind of query mapping is not a quick feature to add. For example, to do this we'll need to have the column info in the `model: ModelDefinition`.